.github/workflows/create-token-pr.yaml

```diff
@@ -0,0 +1,58 @@
+###
+#
+# Creates a PR from the token branch whenever is has commits ahead of main.
+#
+###
+
+name: Create Token PR
+on:
+  push:
+    branches:
+      - tokens/v*
+
+jobs:
+  create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the token branch which was pushed
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Get the name of the token branch which was pushed and the corresponding PR branch
+      - name: Get Branch
+        id: branch-names
+        run: |
+          echo "token-branch=tokens/$(echo ${GITHUB_REF##*/})"    >> $GITHUB_OUTPUT
```

Comment:
Wouldn't that be the same as `${{ github.ref_name }}` (which by the way is provided globally in a workflow)?
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context

With this, you may be able to reduce on step.